[2021-09-02T03:20:26.200315] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `io.xeros.sql.DatabaseCredentials` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('[{"url":"jdbc:mysql://127.0.0.1/u382793551_Xeros?useSSL=false&serverTimezone=UTC", "username":"root", "password":"jjoshua", }]')
 at [Source: (File); line: 3, column: 17] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:20:26.2033177] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:63)
[2021-09-02T03:20:26.2033177] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1429)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1059)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.ValueInstantiator._createFromStringFallbacks(ValueInstantiator.java:371)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.std.StdValueInstantiator.createFromString(StdValueInstantiator.java:323)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromString(BeanDeserializerBase.java:1373)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:171)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:20:26.2043186] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:20:26.2053195] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:20:26.2053195] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:20:26.2053195] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:20:26.2053195] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:25:17.4218878] com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "url" (class io.xeros.ServerConfiguration), not marked as ignorable (6 known properties: "embedded_password", "server_state", "hiscores_database", "store_database", "local_database", "vote_database"])
 at [Source: (File); line: 4, column: 79] (through reference chain: io.xeros.ServerConfiguration["url"])
[2021-09-02T03:25:17.4238897] 	at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61)
[2021-09-02T03:25:17.4238897] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:840)
[2021-09-02T03:25:17.4238897] 	at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:1192)
[2021-09-02T03:25:17.4238897] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1592)
[2021-09-02T03:25:17.4238897] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1570)
[2021-09-02T03:25:17.4238897] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:294)
[2021-09-02T03:25:17.4248912] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:25:17.4248912] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:25:17.4248912] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:25:17.4248912] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:25:17.4248912] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:25:17.4248912] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:25:17.4248912] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:26:14.1058908] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `io.xeros.sql.DatabaseCredentials` out of START_ARRAY token
 at [Source: (File); line: 4, column: 1] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1442)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1216)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1168)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromArray(BeanDeserializerBase.java:1460)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:185)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:26:14.1088936] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:26:14.1098946] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:26:14.1098946] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:26:14.1098946] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:26:14.1098946] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:26:14.1098946] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:26:14.1098946] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:26:36.6403604] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `io.xeros.sql.DatabaseCredentials` out of START_ARRAY token
 at [Source: (File); line: 3, column: 17] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:26:36.6423624] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1442)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1216)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1168)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromArray(BeanDeserializerBase.java:1460)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:185)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:26:36.643363] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:26:36.643363] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:26:36.644364] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:26:36.644364] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:26:36.644364] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:27:57.8891927] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `io.xeros.sql.DatabaseCredentials` out of START_ARRAY token
 at [Source: (File); line: 3, column: 17] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:27:57.8911949] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
[2021-09-02T03:27:57.8911949] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1442)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1216)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1168)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromArray(BeanDeserializerBase.java:1460)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:185)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:27:57.8921958] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:27:57.8921958] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:27:57.8921958] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:27:57.8921958] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:27:57.8931965] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:32:43.9745603] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `io.xeros.sql.DatabaseCredentials` out of START_ARRAY token
 at [Source: (File); line: 4, column: 3] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:32:43.9765624] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
[2021-09-02T03:32:43.9765624] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1442)
[2021-09-02T03:32:43.9765624] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1216)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1168)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromArray(BeanDeserializerBase.java:1460)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:185)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:32:43.9775633] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:32:43.9775633] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:32:43.9775633] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:32:43.9775633] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:32:43.9775633] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:33:42.3405787] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `io.xeros.sql.DatabaseCredentials` out of START_ARRAY token
 at [Source: (File); line: 4, column: 3] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:33:42.3425808] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
[2021-09-02T03:33:42.3425808] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1442)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1216)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1168)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromArray(BeanDeserializerBase.java:1460)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:185)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:33:42.3435817] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:33:42.3435817] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:33:42.3435817] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:33:42.3435817] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:33:42.3445828] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:35:25.0083987] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `io.xeros.sql.DatabaseCredentials` out of START_ARRAY token
 at [Source: (File); line: 3, column: 17] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1442)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1216)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1168)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromArray(BeanDeserializerBase.java:1460)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:185)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:35:25.0104008] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:35:25.011402] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:35:25.011402] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:35:25.011402] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:35:25.011402] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:35:25.011402] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:35:25.011402] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:35:25.011402] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:35:38.637788] com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot deserialize instance of `io.xeros.sql.DatabaseCredentials` out of START_ARRAY token
 at [Source: (File); line: 3, column: 17] (through reference chain: io.xeros.ServerConfiguration["local_database"])
[2021-09-02T03:35:38.6397897] 	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:59)
[2021-09-02T03:35:38.6397897] 	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1442)
[2021-09-02T03:35:38.6397897] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1216)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.DeserializationContext.handleUnexpectedToken(DeserializationContext.java:1168)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromArray(BeanDeserializerBase.java:1460)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeOther(BeanDeserializer.java:185)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:161)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.deser.impl.FieldProperty.deserializeAndSet(FieldProperty.java:138)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:288)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:151)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4202)
[2021-09-02T03:35:38.6407906] 	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3070)
[2021-09-02T03:35:38.6407906] 	at io.xeros.util.JsonUtil.fromYaml(JsonUtil.java:124)
[2021-09-02T03:35:38.6407906] 	at io.xeros.Server.loadConfiguration(Server.java:225)
[2021-09-02T03:35:38.6407906] 	at io.xeros.Server.lambda$main$0(Server.java:165)
[2021-09-02T03:35:38.6407906] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T03:37:24.1651667] org.flywaydb.core.api.FlywayException: Found non-empty schema(s) `u382793551_xeros` but no schema history table. Use baseline() or set baselineOnMigrate to true to initialize the schema history table.
[2021-09-02T03:37:24.1671686] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:202)
[2021-09-02T03:37:24.1671686] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:172)
[2021-09-02T03:37:24.1671686] 	at org.flywaydb.core.Flyway.execute(Flyway.java:588)
[2021-09-02T03:37:24.1671686] 	at org.flywaydb.core.Flyway.migrate(Flyway.java:172)
[2021-09-02T03:37:24.1671686] 	at io.xeros.Server.loadData(Server.java:136)
[2021-09-02T03:37:24.1671686] 	at io.xeros.Server.lambda$main$0(Server.java:172)
[2021-09-02T03:37:24.1671686] 	at io.xeros.GameThread.run(GameThread.java:52)
Sep 02, 2021 3:38:30 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 3:38:31 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 3:38:31 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 3:38:31 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 3:38:31 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 3:38:31 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 3:38:31 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 3:38:37 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 3:38:37 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 3:38:37 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 233 commands.
Sep 02, 2021 3:38:37 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 3:38:37 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
Sep 02, 2021 3:46:41 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 3:46:41 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 3:46:41 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 3:46:41 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 3:46:41 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 3:46:41 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 3:46:41 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 3:46:47 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 3:46:47 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 3:46:47 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 233 commands.
Sep 02, 2021 3:46:47 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 3:46:47 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
Sep 02, 2021 4:04:17 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 4:04:17 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 4:04:17 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 4:04:17 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 4:04:17 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 4:04:17 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 4:04:17 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 4:04:23 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 4:04:23 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 4:04:23 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 234 commands.
Sep 02, 2021 4:04:23 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 4:04:23 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
[2021-09-02T04:48:47.9385639] org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Detected applied migration not resolved locally: 12. If you removed this migration intentionally, run repair to mark the migration as deleted.
Need more flexibility with validation rules? Learn more: https://flywaydb.org/custom-validate-rules/?ref=v7.11.0
[2021-09-02T04:48:47.9405657] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:179)
[2021-09-02T04:48:47.9405657] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:172)
[2021-09-02T04:48:47.9405657] 	at org.flywaydb.core.Flyway.execute(Flyway.java:588)
[2021-09-02T04:48:47.9415667] 	at org.flywaydb.core.Flyway.migrate(Flyway.java:172)
[2021-09-02T04:48:47.9415667] 	at io.xeros.Server.loadData(Server.java:136)
[2021-09-02T04:48:47.9415667] 	at io.xeros.Server.lambda$main$0(Server.java:172)
[2021-09-02T04:48:47.9415667] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T04:53:44.1051747] org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Detected applied migration not resolved locally: 12. If you removed this migration intentionally, run repair to mark the migration as deleted.
Need more flexibility with validation rules? Learn more: https://flywaydb.org/custom-validate-rules/?ref=v7.11.0
[2021-09-02T04:53:44.107177] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:179)
[2021-09-02T04:53:44.107177] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:172)
[2021-09-02T04:53:44.107177] 	at org.flywaydb.core.Flyway.execute(Flyway.java:588)
[2021-09-02T04:53:44.107177] 	at org.flywaydb.core.Flyway.migrate(Flyway.java:172)
[2021-09-02T04:53:44.107177] 	at io.xeros.Server.loadData(Server.java:136)
[2021-09-02T04:53:44.107177] 	at io.xeros.Server.lambda$main$0(Server.java:172)
[2021-09-02T04:53:44.107177] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T04:53:56.4363762] org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Detected applied migration not resolved locally: 12. If you removed this migration intentionally, run repair to mark the migration as deleted.
Need more flexibility with validation rules? Learn more: https://flywaydb.org/custom-validate-rules/?ref=v7.11.0
[2021-09-02T04:53:56.4393788] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:179)
[2021-09-02T04:53:56.4393788] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:172)
[2021-09-02T04:53:56.4393788] 	at org.flywaydb.core.Flyway.execute(Flyway.java:588)
[2021-09-02T04:53:56.4393788] 	at org.flywaydb.core.Flyway.migrate(Flyway.java:172)
[2021-09-02T04:53:56.4393788] 	at io.xeros.Server.loadData(Server.java:136)
[2021-09-02T04:53:56.4393788] 	at io.xeros.Server.lambda$main$0(Server.java:172)
[2021-09-02T04:53:56.4393788] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T04:54:04.9551147] org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Detected applied migration not resolved locally: 12. If you removed this migration intentionally, run repair to mark the migration as deleted.
Need more flexibility with validation rules? Learn more: https://flywaydb.org/custom-validate-rules/?ref=v7.11.0
[2021-09-02T04:54:04.9581172] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:179)
[2021-09-02T04:54:04.9581172] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:172)
[2021-09-02T04:54:04.9581172] 	at org.flywaydb.core.Flyway.execute(Flyway.java:588)
[2021-09-02T04:54:04.9581172] 	at org.flywaydb.core.Flyway.migrate(Flyway.java:172)
[2021-09-02T04:54:04.9581172] 	at io.xeros.Server.loadData(Server.java:136)
[2021-09-02T04:54:04.9591184] 	at io.xeros.Server.lambda$main$0(Server.java:172)
[2021-09-02T04:54:04.9591184] 	at io.xeros.GameThread.run(GameThread.java:52)
[2021-09-02T04:54:33.7242474] org.flywaydb.core.api.exception.FlywayValidateException: Validate failed: Migrations have failed validation
Migration description mismatch for migration version 12
-> Applied to database : create leaderboard tables
-> Resolved locally    : clear calendar tables for new month. Either revert the changes to the migration, or run repair to update the schema history.
Need more flexibility with validation rules? Learn more: https://flywaydb.org/custom-validate-rules/?ref=v7.11.0
[2021-09-02T04:54:33.7262493] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:179)
[2021-09-02T04:54:33.7262493] 	at org.flywaydb.core.Flyway$1.execute(Flyway.java:172)
[2021-09-02T04:54:33.7262493] 	at org.flywaydb.core.Flyway.execute(Flyway.java:588)
[2021-09-02T04:54:33.7272504] 	at org.flywaydb.core.Flyway.migrate(Flyway.java:172)
[2021-09-02T04:54:33.7272504] 	at io.xeros.Server.loadData(Server.java:136)
[2021-09-02T04:54:33.7272504] 	at io.xeros.Server.lambda$main$0(Server.java:172)
[2021-09-02T04:54:33.7272504] 	at io.xeros.GameThread.run(GameThread.java:52)
Sep 02, 2021 4:55:14 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 4:55:15 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 4:55:15 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 4:55:15 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 4:55:15 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 4:55:15 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 4:55:15 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 4:55:21 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 4:55:21 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 4:55:21 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 234 commands.
Sep 02, 2021 4:55:22 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 4:55:22 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
Sep 02, 2021 5:04:57 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 5:04:57 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 5:04:57 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 5:04:57 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 5:04:58 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 5:04:58 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 5:04:58 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 5:05:03 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 5:05:03 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 5:05:03 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 234 commands.
Sep 02, 2021 5:05:04 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 5:05:04 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
[2021-09-02T05:05:58.8962532] java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
[2021-09-02T05:05:58.8962532] 	at io.xeros.content.leaderboards.LeaderboardInterface.handleButtons(LeaderboardInterface.java:41)
[2021-09-02T05:05:58.8962532] 	at io.xeros.model.entity.player.packets.ClickingButtons.processPacket(ClickingButtons.java:160)
[2021-09-02T05:05:58.897254] 	at io.xeros.model.entity.player.PacketHandler.processPacket(PacketHandler.java:35)
[2021-09-02T05:05:58.897254] 	at io.xeros.model.entity.player.Player.processQueuedPackets(Player.java:2871)
[2021-09-02T05:05:58.897254] 	at io.xeros.model.entity.player.PlayerHandler.lambda$process$4(PlayerHandler.java:301)
[2021-09-02T05:05:58.897254] 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
[2021-09-02T05:05:58.897254] 	at io.xeros.model.entity.player.PlayerHandler.process(PlayerHandler.java:298)
[2021-09-02T05:05:58.897254] 	at io.xeros.GameThread.lambda$setTickables$2(GameThread.java:29)
[2021-09-02T05:05:58.897254] 	at io.xeros.GameThread.tick(GameThread.java:42)
[2021-09-02T05:05:58.897254] 	at io.xeros.GameThread.run(GameThread.java:56)
[2021-09-02T05:05:59.4957976] java.lang.ArrayIndexOutOfBoundsException: Index 2 out of bounds for length 1
[2021-09-02T05:05:59.4957976] 	at io.xeros.content.leaderboards.LeaderboardInterface.handleButtons(LeaderboardInterface.java:41)
[2021-09-02T05:05:59.4957976] 	at io.xeros.model.entity.player.packets.ClickingButtons.processPacket(ClickingButtons.java:160)
[2021-09-02T05:05:59.4957976] 	at io.xeros.model.entity.player.PacketHandler.processPacket(PacketHandler.java:35)
[2021-09-02T05:05:59.4957976] 	at io.xeros.model.entity.player.Player.processQueuedPackets(Player.java:2871)
[2021-09-02T05:05:59.4957976] 	at io.xeros.model.entity.player.PlayerHandler.lambda$process$4(PlayerHandler.java:301)
[2021-09-02T05:05:59.4957976] 	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
[2021-09-02T05:05:59.4957976] 	at io.xeros.model.entity.player.PlayerHandler.process(PlayerHandler.java:298)
[2021-09-02T05:05:59.4957976] 	at io.xeros.GameThread.lambda$setTickables$2(GameThread.java:29)
[2021-09-02T05:05:59.4957976] 	at io.xeros.GameThread.tick(GameThread.java:42)
[2021-09-02T05:05:59.4957976] 	at io.xeros.GameThread.run(GameThread.java:56)
Sep 02, 2021 5:06:31 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 5:06:31 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 5:06:31 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 5:06:31 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 5:06:31 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 5:06:31 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 5:06:31 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 5:06:37 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 5:06:37 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 5:06:37 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 234 commands.
Sep 02, 2021 5:06:37 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 5:06:37 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
Sep 02, 2021 5:10:11 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 5:10:12 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 5:10:12 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 5:10:12 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 5:10:12 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 5:10:12 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 5:10:12 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 5:10:18 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 5:10:18 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 5:10:18 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 234 commands.
Sep 02, 2021 5:10:18 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 5:10:18 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
Sep 02, 2021 5:12:02 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 5:12:02 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 5:12:02 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 5:12:02 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 5:12:02 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 5:12:02 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 5:12:02 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 5:12:08 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 5:12:08 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 5:12:08 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 234 commands.
Sep 02, 2021 5:12:09 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 5:12:09 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
Sep 02, 2021 5:15:03 AM io.xeros.content.boosts.Boosts init
INFO: Loaded 16 boosts.
Sep 02, 2021 5:15:03 AM io.xeros.model.definitions.ItemDef load
INFO: Loaded 14045 item definitions.
Sep 02, 2021 5:15:03 AM io.xeros.model.definitions.NpcStats load
INFO: Loaded 2906 npc stats.
Sep 02, 2021 5:15:04 AM io.xeros.model.definitions.ItemStats load
INFO: Loaded 9767 item stats.
Sep 02, 2021 5:15:04 AM io.xeros.model.entity.npc.NpcSpawnLoader load
INFO: Spawned 2925 npcs.
Sep 02, 2021 5:15:04 AM io.xeros.model.entity.npc.NPCRelationship setup
INFO: Initialized 36 npc relationships.
Sep 02, 2021 5:15:04 AM io.xeros.model.lobby.LobbyManager initializeLobbies
INFO: Initialized 2 lobbies.
Sep 02, 2021 5:15:09 AM io.xeros.model.collisionmap.Region load
INFO: Loaded 52 custom maps.
Sep 02, 2021 5:15:09 AM io.xeros.model.collisionmap.Region load
INFO: Error loading map files: [1182, 1183, 3642, 3643, 3648, 3649]
Sep 02, 2021 5:15:10 AM io.xeros.content.commands.CommandManager initializeCommands
INFO: Loaded 234 commands.
Sep 02, 2021 5:15:10 AM io.xeros.content.dailyrewards.DailyRewardContainer load
INFO: Selected daily rewards path with start date: 2021-07-01
Sep 02, 2021 5:15:10 AM io.xeros.content.dailyrewards.DailyRewardsRecords load
WARNING: No daily reward claims found.
